assets/samplesheet_1_row_sample_multi.csv

```diff
@@ -0,0 +1,2 @@
+sample,image_directory,cycle_images
+TEST1,/home/pollen/HITS/nextflow/test_mcmicro-nf-core/input_sample_multi_1,cycif-tonsil-cycle1.ome.tif cycif-tonsil-cycle2.ome.tif
```

Comment:
local path

Reply:
and spaces?

Reply:
The space was a separator for the `cycle_images` column but I've changed that to semicolon now.